tracer/src/Datadog.Trace/Configuration/ConfigurationSources/Telemetry/ConfigurationBuilder.cs

```diff
@@ -10,11 +10,15 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using Datadog.Trace.Configuration.ConfigurationSources.Telemetry;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.Telemetry.Metrics;
 
 namespace Datadog.Trace.Configuration.Telemetry;
 
 internal readonly struct ConfigurationBuilder
 {
+    private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<ConfigurationBuilder>();
```

Comment:
I don't think you can/should reference `Log` in here. The logger configuration requires referencing this which leaves you with a potential circular deps issue 😬

Reply:
> The logger configuration requires referencing this which leaves you with a potential circular deps issue 😬

I did not know this, is it something that we should add as a doc comment to the `DatadogLogger`?

I do think though we need a way to log out when there is both the OpenTelemetry configuration and Datadog configuration where we take the Datadog integration over it

But that can come later IMO

Reply:
FYI I've removed the offending calls to `Log` in e8fc0b590dbb733caa248882d8c6215cdd9a4a57 . As a follow-up I'll figure out the right way to log our diagnostics while keeping the functionality the same